strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/services/ArtifactManagementService.java

```diff
@@ -16,10 +16,10 @@
 public interface ArtifactManagementService extends ConfigurationService
 {
 
-    void store(String storageId,
-               String repositoryId,
-               String path,
-               InputStream is)
+    void validateAndStore(String storageId,
```

Comment:
My apologies! I now see what you mean!
Okay, this is all good now! Thanks a lot! :)